### PR TITLE
Remove insights- repo name

### PR DESCRIPTION
**Pull request by karelhala**
https://issues.redhat.com/browse/RHCLOUD-11596

### Remove insights from repo name

Since we don't want to confuse consumers of this repository, we should remove the insights prefix. This goes hand to hand with renaming all the references in this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,68 +60,16 @@ Note: You will need to set up the Insights environment if you want to develop wi
...
 
 ## Technologies
 
 ### Webpack
 
-#### Webpack.config.js
+There is [shared common config](https://www.npmjs.com/package/@redhat-cloud-services/frontend-components-config) with predefined values and lifecycles to build and run your application.
 
-This file exports an object with the configuration for webpack and webpack dev server.
+We are used [federated modules](https://webpack.js.org/concepts/module-federation/) to seamlessly load multiple application.
```
Comment (Hyperkid123) [inline README.md:71]: We are used -> We use

File: README.md [hunk 2]
```diff
@@ -60,68 +60,16 @@ Note: You will need to set up the Insights environment if you want to develop wi
 
 ## Insights Components
 
-Insights Platform will deliver components and static assets through [npm](https://www.npmjs.com/package/@red-hat-insights/insights-frontend-components). ESI tags are used to import the [chroming](https://github.com/RedHatInsights/insights-chrome) which takes care of the header, sidebar, and footer.
+Insights Platform will deliver components and static assets through [npm](https://www.npmjs.com/package/@redhat-cloud-services/frontend-components). ESI tags are used to import the [chroming](https://github.com/RedHatInsights/insights-chrome) which takes care of the header, sidebar, and footer.
```
Comment (ryelo) [inline README.md:63]: Can we remove `Insights` from each of these?

**Review by ryelo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (karelhala) [reply README.md:63]: Good point, I went trough the docs and removed bunch of other insights mentions.

File: README.md (hunk 1 above)
Comment (karelhala) [reply README.md:71]: I wanted to go with `We are using` and `We used` and end up with `We are used` :smile:

**Review by ryelo — APPROVED** [review]

**Pull request merged**